### PR TITLE
Bugs: general fixes

### PR DESCRIPTION
**Pull request by edbrett**
- css style fixes for participate section (homepage)
- update data feed for community section (homepage)
- fix rendering polygons on front end for tasks and events single views
- update map component to also show pop up the above views
- Reimplement responsive mobile styles for all tables (from old React component)
- Fix the order of the switchers in the maps

Other: added snippet to give smooth scroll to anchor links site wide for improved Homepage experience.

### CODE REVIEW

File: app/assets/javascripts/components/Map.js [hunk 1]
```diff
@@ -75,10 +75,12 @@ class Map extends React.Component {
   }
 
   _popUpFor(layer) {
+    // debugger
     return `<h1>${layer.name}</h1>
             <p>${layer.description}</p>
             <div class="link-group">
-              <a href="/tasks/${layer.id}">Learn more</a>
+              <a href="/campaigns/${layer.events ? layer.events[0].campaign_id : this.props.campaignId}">Campaign</a>
```
Comment (simaob) [inline app/assets/javascripts/components/Map.js:82]: A task might belong to more than one event and/or campaigns, so I think we should maybe iterate over them and display a link to each. Probably worth checking with Juanca for ideas on how best to show it.

**Review by simaob — CHANGES_REQUESTED** [review]
Hi Edward, most things are ticked, but there's a comment to address here and some others on Pivotal Tracker. Let me know if something is not clear! Happy to discuss, hopefully Wednesday I'll have more time than I did on Monday! =) Thanks!

File: app/assets/javascripts/components/Map.js (hunk 1 above)
Comment (edbrett) [reply app/assets/javascripts/components/Map.js:82]: I have discussed with Juan Carlos and will try to find a design solution. With regards to responsive maps fitting to screen height, we have the use case for events that there is a header above the map therefore on mobile map fitting is not possible. I can approximate but not force map to fit to bottom of screen in all cases.

**Comment by edbrett** [comment]
Updated

File: app/assets/javascripts/components/Map.js (hunk 1 above)
Comment (simaob) [reply app/assets/javascripts/components/Map.js:82]: That works for me! =) Thanks!

**Review by simaob — APPROVED** [review]

**Pull request merged**